src/Task.ts

```diff
@@ -2,7 +2,10 @@ import { Address, Bytes, log } from '@graphprotocol/graph-ts'
 
 import { Task } from '../types/schema'
 
-import { BalanceConnectorsSet } from '../types/templates/Task/Task'
+import { BalanceConnectorsSet,
+  TimeLockDelaySet,
+  TimeLockExecutionPeriodSet,
+  TimeLockExpirationSet } from '../types/templates/Task/Task'
```

Comment:
Not sure why the linter is not picking this one :/

```suggestion
import { 
  BalanceConnectorsSet,
  TimeLockDelaySet,
  TimeLockExecutionPeriodSet,
  TimeLockExpirationSet 
} from '../types/templates/Task/Task'
```